tremor-connectors/src/utils/tls.rs

```diff
@@ -216,167 +204,57 @@ pub enum Error {
 }
 
 /// Load the passed certificates file
-fn load_certs(path: &Path) -> Result<Vec<Certificate>, Error> {
-    let certfile = tremor_common::file::open(path)?;
+fn load_certs<'x>(path: &PathBuf) -> Result<Vec<CertificateDer<'x>>, Error> {
```

Comment:
If we can keep this path I think it;ll reduce changes below

Reply:
I don't think it will. The lifetime is a side-effect of changes in rustls_pki_types for `pub struct CertificateDer<'a>(Der<'a>)`